packages/drivers/driver-base/package-lock.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@fluidframework/driver-base",
```

Comment:
This adds package locks for packages that didn't have them before. We should look into why.